modules/ingester/instance.go

```diff
@@ -337,15 +345,11 @@ func (i *instance) getOrCreateTrace(req *tempopb.PushRequest) (*trace, error) {
 		return trace, nil
 	}
 
-	err = i.limiter.AssertMaxTracesPerUser(i.instanceID, len(i.traces))
-	if err != nil {
-		return nil, status.Errorf(codes.FailedPrecondition, "%s max live traces per tenant exceeded: %v", overrides.ErrorPrefixLiveTracesExceeded, err)
-	}
-
 	maxBytes := i.limiter.limits.MaxBytesPerTrace(i.instanceID)
 	trace = newTrace(maxBytes, fp, traceID)
 	i.traces[fp] = trace
 	i.tracesCreatedTotal.Inc()
```

Comment:
nit: can we metric this off the atomic int?

Reply:
i think this is fine. this is metric'ed by the distributor as refused.